plugins/node/opentelemetry-instrumentation-mysql/src/types.ts

```diff
@@ -16,4 +16,10 @@
 
 import { InstrumentationConfig } from '@opentelemetry/instrumentation';
 
-export type MySQLInstrumentationConfig = InstrumentationConfig;
+export interface MySQLInstrumentationConfig extends InstrumentationConfig {
+  /**
+   * If true, an attribute containing the query's parameters will be attached
+   * the spans generated to represent the query.
```

Comment:
nit: could be great to align this text with the one in the README :)

Reply:
done